src/callbacks/onResize/annotateGrouping/horizontally.js

```diff
@@ -14,11 +14,23 @@ export default function horizontally() {
                     .classed('annotation', true)
                     .attr({
                         x: 0,
-                        dx: -this.margin.left,
                         y: y1,
-                        dy: this.y.rangeBand() * 1.75
+                        dy: this.y.rangeBand() * 1.25
                     })
-                    .text(d.key);
+                    .text(
+                        `${this.config.groupings.filter(
+                            grouping => grouping.value_col === this.config.y.grouping
+                        )[0].label}: ${d.key}`
+                    ),
+                rule = g
```

Comment:
Don't think the rule is needed with the rest of the refactor. I'll take it out in a sec.

Reply:
Actually - I'm just going to make an issue. I'm getting formatting differences when I rebuild, and don't want to introduce those right now ...

Reply:
#115